api/napi/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+#![deny(clippy::all)]
+
+use slint_interpreter::{ComponentCompiler, ComponentHandle};
+
+mod interpreter;
+pub use interpreter::*;
+
+mod types;
+pub use types::*;
+
+#[macro_use]
+extern crate napi_derive;
```

Comment:
I think i prefer stuff like `use napi_derive::napi;`
Is there a reason you did it like this?

Reply:
I don't like it. It was on the example and I have trouble with the  `[napi]` macro without it by just using `use napi_derive::napi`.